src/routes/components/index.svelte

```diff
@@ -46,6 +46,8 @@
 		.sort(compare(sorting));
 	$: categories = extractUnique(dataToDisplay, 'category');
 	$: filterTag = selectedTags?.map((obj) => obj.value) || [];
+
+	const categoryId = { 'SvelteKit Adapters': 'adapters' };
```

Comment:
Let's add an ID for each:
```suggestion
	const categoryId = {
		'Animations': 'animations',
		'Data Visualisation': 'data-vis',
		'Design Pattern': 'design-patterns',
		'Design System': 'design-systems',
		'Developer Experience': 'dx',
		'Forms & User Input': 'input',
		'Integration': 'integrations',
		'Rich Text Editor': 'text-editors',
		'Routers': 'routers',
		'Stores': 'stores',
		'SvelteKit Adapters': 'adapters',
		'Testing': 'testing',
		'Unclassified': 'unclassified',
		'User Interaction': 'ui'
	};
```